fog/load_testing/src/bin/ingest.rs

```diff
@@ -220,6 +262,21 @@ fn load_test(ingest_server_binary: &Path, test_params: TestParams, logger: Logge
             );
         }
 
+        // Tell server to activate
+        {
+            let ingest_client = FogIngestGrpcClient::new(
+                client_listen_uri,
+                Duration::from_secs(30),
+                grpcio_env,
+                logger.clone(),
+            );
+            ingest_client
+                .activate()
+                .expect("Could not activate ingest server");
+            std::thread::sleep(std::time::Duration::from_millis(10));
+            ingest_server.assert_not_stopped();
+        }
```

Comment:
this is needed because this code hasn't been touched since the ingest server was required to be activating it, and nothing was exercising the load test